### DISCUSSION TITLE
阿里云盘webdav无法上传

### DISCUSSION THREAD

**Discussion by cvpas** [AlistGo/alist#834] [Q&A]
使用docker搭建alist后，添加阿里云盘账号，无proxy设置，也没开启反代，元信息也开启上传功能。
发现macOS使用自带的finder连接服务器后，能下载上面的文件，也能重命名，但是无法上传和复制文件，登录alist后台会发现有个同名“假”文件（以“.”开头），alist网页端可以正常上传。请问这是什么原因呢？
<img width="749" alt="截屏2022-03-28 上午10 43 07" src="https://user-images.githubusercontent.com/36500518/160317958-ca49b46d-bb5f-48ae-be8f-5739165df664.png">
<img width="1050" alt="截屏2022-03-28 上午10 43 32" src="https://user-images.githubusercontent.com/36500518/160317964-39870dad-3a25-4886-88c6-24f03aa40dc9.png">

**Comment by xhofe** [comment]
尽量使用其他的webdav软件而不是原生的，他们通常适配的不好

**Comment by cvpas** [comment]
ios文件app也是连接正常，但是无法访问文件夹内容
![WechatIMG13](https://user-images.githubusercontent.com/36500518/160318735-063bf7e7-c930-4055-bf56-c5bf1a04032d.jpeg)

**Comment by cvpas** [comment]
主要使用场景是macOS和iOS上的finder和文件app，请问有无可能增强这两个连接的适配呢，谢谢！

**Comment by xhofe** [comment]
无此计划，开发时只会对CloudMounter进行测试。

**Reply by cvpas** [reply]
> 无此计划，开发时只会对CloudMounter进行测试。
下载cloudmounter挂载阿里云盘后发现还是上传错误，出现如下提示
<img width="372" alt="截屏2022-03-29 下午8 51 27" src="https://user-images.githubusercontent.com/36500518/160615731-e4aa4ee5-7617-4f2b-8c14-fd4025569c17.png">

**Reply by xhofe** [reply]
我测试使用CloudMounter无任何问题。
